docs/components/SegmentedControl/Web.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import { Meta, StoryFn } from "@storybook/react";
+import { SegmentedControl } from "@jobber/components/SegmentedControl";
+
+const meta: Meta = {
+  title: "Components/Selections/SegmentedControl/Web",
+  component: SegmentedControl,
+  parameters: {
+    viewMode: "story",
+    previewTabs: {
+      code: {
+        hidden: false,
+      },
+    },
+  },
+};
+export default meta;
+
+type Story = StoryFn<typeof SegmentedControl>;
+const options = [
+  { value: "pizza", label: "Pizza" },
+  { value: "tacos", label: "Tacos" },
+  { value: "sushi", label: "Sushi" },
+  { value: "burgers", label: "Burgers" },
+];
+
+export const Controlled: Story = () => {
+  const [activeOption, setActiveOption] = useState("tacos");
+
+  return (
+    <SegmentedControl
```

Comment:
If I don't look in the underlying code and only look on Storybook these two examples are the same right? Can we modify them so that the controlled and uncontrolled behaviour is clear while only looking in Storybook?